py3-propcache.yaml

```diff
@@ -0,0 +1,84 @@
+package:
+  name: py3-propcache
+  version: 0.2.0
+  epoch: 0
+  description: Fast property caching
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provider-priority: 0
+
+vars:
+  pypi-package: propcache
+
+data:
+  - name: py-versions
+    items:
+      3.10: '310'
+      3.11: '311'
+      3.12: '312'
+      3.13: '300'
```

Comment:
the `3.13: '300' entry here is correct and consistent with all our other multi-versioned python packages in wolfi.